changelog.txt

```diff
@@ -1,5 +1,7 @@
 0.81
  * Fixed bug where creating a Realm instance with a hashcode collision no longer returned the wrong Realm instance.
+ * Updated Realm Core to version 0.89.1
+   - Fixed bug in "index rebuilding" (would delete the wrong column, causing crash).
```

Comment:
We should mention updates of Realm Core, right? And when #1038 is merged, this bug fix is important.